setup.py

```diff
@@ -28,7 +31,8 @@
     packages=setuptools.find_packages(),
     install_requires=install_requires,
     extras_require={
-        'napari': ['napari-gui==0.0.5.1', 'matplotlib==2.1.2']
+        'napari': ['napari-gui==0.0.5.1', 'matplotlib==2.1.2'],
```

Comment:
with the loose dependencies, matlab does not need to be pinned here

Reply:
Is that true?  I thought `napari` requires this specific version of matplotlib.  @kne42?

Reply:
We don't require any versions of matplotlib. I think it was some vispy thing which @shanaxel42 and @ambrosejcarr discovered. So I can't tell you for sure.

Reply:
@neuromusic napari 0.0.5.1 requires matplotlib<2.2.2 via their vispy dependency. To my knowledge, this relationship is required until that is fixed.

Reply:
I cannot replicate the incompatibility I have napari installed with a more recent matplotlib, no problem

```
>>> import napari_gui
>>> napari_gui.__version__
'0.0.5.1'
>>> import matplotlib
>>> matplotlib.__version__
'3.0.2'
>>> 
```